lib/modelManagers/articlemodel.js

```diff
@@ -108,7 +116,96 @@ class articleModelManager {
     const returnedArticles = await Articles.findAll(options);
     return returnedArticles;
   }
-}
 
+  /**
+ *
+ *
+ * @static
+ * @param {where} where
+ * @param {value} value
+ * @param {limit} limit
+ * @param {offset} offset
+ * @memberof articleModelManager
+ * @returns {object} Articles by title
+ */
+  static async getArticlesByTitle(where, value, limit, offset) {
+    const options = {
+      include: [includeUser],
+      limit,
+      offset
+    };
+    options.where = where;
+    options.attributes = searchAttribute.attributes;
+    const returnedArticles = await Articles.findAll(options);
+    return returnedArticles;
+  }
+
+  /**
+ *
+ *
+ * @static
+ * @param {where} where
+ * @param {value} value
+ * @param {limit} limit
+ * @param {offset} offset
+ * @memberof articleModelManager
+ * @returns {object} Articles by author
+ */
+  static async getArticlesByAuthor(where, value, limit, offset) {
+    const options = {
+      include: [{ ...includeUser, where }],
```

Comment:
It is the same reason as above. I just believe it is not needed since, when the user clicks on the article of choice from the search result, every other details will be included.

Reply:
For consistency sake including tags in all `get articles` functions is a better way to go. We can't have tags in some functions and not have it in others